lib/knetwork/nopclose.go

```diff
@@ -0,0 +1,35 @@
+package knetwork
+
+import (
+	"io"
+)
+
+type nopReadCloser struct {
```

Comment:
This already exists in the io package? https://golang.org/src/io/io.go?s=6022:6080#L612

Reply:
removed, good find.